model/resource/host.yaml

```diff
@@ -70,3 +70,12 @@ groups:
           The version string of the VM image or host OS as defined in
           [Version Attributes](README.md#version-attributes).
         examples: ['0.1']
+      - id: ip
```

Comment:
we avoided using `IP` in the server/client namespaces and used `address`  as it allows covering other kinds of socket addresses. I'd propose to use `host.socket.address` here as well.

Reply:
In the context of a particular operation, only one kind of address will be relevant. But a host will have multiple address kinds which are not all distinguishable by syntax. So I think ip is fine in the name.

Reply:
I agree with @Oberon00 's comment above.

Plus, I assume in the overwhelming majority of cases the addresses users care about would be IP. So, from a user perspective `host.socket.address` would be not the most intuitive attribute to search for when someone would want to know the IP addresses of a host.

Instead, I'd propose to add additional fields for other families of socket addresses, if we want to model them on the host resource. But let's do it based on practical examples / use cases and a based on a concrete need.